lua/DamageIndicator.lua

```diff
@@ -94,13 +94,15 @@ elseif string.lower(RequiredScript) == "lib/units/beings/player/playerdamage" th
 
 	function PlayerDamage:damage_fire(attack_data, ...)
 		local value = PlayerDamage_damage_fire(self, attack_data, ...)
-		if alive(self._unit) and (attack_data.position or attack_data.col_ray.position) then
-			local distance = mvector3.distance(attack_data.position or attack_data.col_ray.position, self._unit:position())
-			if self:_chk_can_take_dmg() and distance <= attack_data.range and not (self._god_mode or self._invulnerable or self._mission_damage_blockers.invulnerable or self:incapacitated() or self._bleed_out) then
-				self:_hit_direction(attack_data.position, HUDHitDirection.DAMAGE_TYPES.FRIENDLY_FIRE)
+		if distnace not nil and attack_data.range not nil then
```

Comment:
I think there is a typo here: `distnace` should be spelled `distance`.
And I think it is useless to check if distance is nil, as it is affected a value line 99.
All in all, if you didn't get any error with a mispelled `distnace` variable, then it confirms that this part is useless, and you can keep only `attack_data.range not nil`